docs/content/docs/build/releases-and-apitags.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Fluid Releases and API Tags
+aliases:
+  - "/build/releases/"
+---
+
+
+## Fluid Release Versions
+
+Fluid Releases will be done through three channels
+
+-   Production
+-   Pre-release
+-   Dev release
+
+### Production Channel
+
+Prod releases are our most stable releases, follow semver and are ready to be used in Production scenarios. These releases correspond to the `latest` tag on npm. This should be the channel used to get the most stable version of Fluid. Following **semver (X.Y.Z)**, this channel will includes:
+
+-   **Major releases (X)** - Used when releasing `breaking changes`. Breaking changes can include API changes, runtime behavior changes and document format changes. Variable release cadence
+-   **Minor releases (Y)** - Used when adding `new features or non-critical bug fixes`, released ~weekly
+-   **Patch releases (Z)** - Used when releasing fixes for `critical, time-sensitive bugs`. Variable release cadence
```

Comment:
Trailing periods, since each bullet point is more than one sentence.
```suggestion
-   **Major releases (X)** - Used when releasing `breaking changes`. Breaking changes can include API changes, runtime behavior changes and document format changes. Variable release cadence.
-   **Minor releases (Y)** - Used when adding `new features or non-critical bug fixes`, released ~weekly.
-   **Patch releases (Z)** - Used when releasing fixes for `critical, time-sensitive bugs`. Variable release cadence.
```